src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildListener.java

```diff
@@ -32,7 +34,7 @@ public void onStarted(AbstractBuild<?, ?> abstractBuild, TaskListener listener)
     try {
       abstractBuild.setDescription(cause.getShortDescription());
     } catch (IOException e) {
-      logger.log(Level.SEVERE, "Can't update build description", e);
+      listener.getLogger().println(format("Can't update build description: %s", e));
```

Comment:
I think we are missing the stacktrace now. Can we print the stacktrace to build logger too?

Reply:
OK, will do.

Reply:
Done.